entities.c

```diff
@@ -390,3 +390,32 @@ size_t decode_html_entities_utf8(char *dest, const char *src)
 	return (size_t)(to - dest);
 }
 
+int encode_html_entities(char *dest, const char *src) {
```

Comment:
Please return `sitze_t` instead of `int`

Reply:
Completely agree. I'll commit the correction.